php_phongo.c

```diff
@@ -632,10 +633,12 @@ bool phongo_execute_write(mongoc_client_t *client, const char *namespace, php_ph
 		return false;
 	}
 
+	opts = bson_new();
```

Comment:
Where is this freed? Also, AFAIK this will always allocate memory. Using `BSON_INITIALIZER` and calling `bson_destroy()` later allows us to use a stack-allocated buffer within this function.